resources/conditions.ts

```diff
@@ -2,14 +2,14 @@
 // are also functions.
 
 import { RaidbossData as Data } from '../types/data';
-import { TargetedMatches } from '../types/trigger';
+import { MatchesAny, TargetedMatches } from '../types/trigger';
 
 export default {
   targetIsYou(): (data: Data, matches: TargetedMatches) => boolean {
-    return (data: Data, matches: TargetedMatches) => data.me === matches?.target;
+    return (data: Data, matches: MatchesAny) => 'target' in matches && data.me === matches.target;
```

Comment:
Is `MatchesAny` just to be safe if the wrong type gets used here?

For what it's worth, I'm not sure either the above `TargetedMatches` adds any type safety here.  I wonder a little if this should just be `MatchesAny`.  It's not clear to me if it is possible to get popuptext to care about matches types here.